test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -23,7 +23,7 @@
         <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
         <PackageReference Include="NUnit.Analyzers" Version="3.3.0" />
         <PackageReference Include="coverlet.collector" Version="3.1.2" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" ExcludeAssets="none" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
```

Comment:
This would remove the extra files inclusion of Microsoft.TestPlatform Nuget related. Need to wait for PR checks to validate if @pinzart90 did this intentionally at https://github.com/DynamoDS/Dynamo/commit/9f63101ce54374fcf33a5df0ee1f7bf7a5ca4756#diff-21ce5a5483bb0007fea169e57cac128c92d3a61ea165cded9b4f3af26f179e67
![image](https://github.com/DynamoDS/Dynamo/assets/3942418/f0af22e3-d30b-43dc-a453-ee583b249ba1)

Reply:
Make sure all tests are still runnable from visua studio editor too.
If we skip the exclude=“none” attribute the all dlls from the package will not be copied to the bin folder(default behavior for tests projects)

Reply:
Also why are we packaging test projects and their dependencies ? Are users of the dynamo package supposed to be able to just run tests on it? It is a bit unusual

Reply:
> Make sure all tests are still runnable from visua studio editor too. If we skip the exclude=“none” attribute the all dlls from the package will not be copied to the bin folder(default behavior for tests projects)

Understood, basically I am trying to see what is the best way to put all test binaries away from any Nuget we are shipping to consumers

Reply:
One way I am thinking is to actually create a new solution to exclude all the testing projects only for build purpose like the service core and our core runtime. Still thinking about it